eng/pipelines/official/stages/publish.yml

```diff
@@ -51,3 +46,14 @@ stages:
         -TsaRepositoryName "$(TsaRepositoryName)"
         -TsaCodebaseName "$(TsaCodebaseName)"
         -TsaPublish $True
+    # Allow symbol publish to emit expected warnings without failing the build. Include single
```

Comment:
What kind of expected warnings are we seeing during symbol publishing?

Reply:
No idea...this was in there before.  I'll check some recent runs.